zha-network-card.js

```diff
@@ -350,6 +382,17 @@ class ZHANetworkCard extends HTMLElement {
       });
   }
 
+  set hass(hass) {
+    console.log("DEBUG: set hass(hass) is called")
+    const config = this._config;
+    const root = this.shadowRoot;
+
+    if (!this.initialized) {
```

Comment:
This is temporary to avoid hammering the zigbee devices with actual cluster attribute requests as `set hass(hass)` is invoked very often.